### PR TITLE
Cleanup job

### PR DESCRIPTION
**Pull request by vilit1**
simple bash script to get everything in the test rg without the exclusions and kill them (if possible)

will run at 5am pst daily to give time for the weekly run to do it's thing

example run: https://dev.azure.com/azureiotdevxp/aziotcli/_build/results?buildId=7351&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=73aacd12-d9e4-5437-3109-ca9de3b050de

note that there is an ADU instance that refuses to be deleted

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

File: .azure-devops/cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Run daily at 5 am (Pacific).
+schedules:
+- cron: "0 13 * * *"
+  displayName: 'Daily Cleanup'
+  branches:
+    include:
+    - dev
+  always: true
+
+parameters:
```
Comment (digimaun) [inline .azure-devops/cleanup.yml:17]: we should refactor this to pull desired RG and exclusions from our variable library, take a look at  https://github.com/Azure/azure-iot-cli-extension/blob/dev/.azure-devops/create-release.yml#L75 for an example pattern, then the parameters become the name of the variable in the var group that has the respective values

Comment (vilit1) [reply .azure-devops/cleanup.yml:17]: but then we need another variable group for this...

ok will do

also will need to understand the difference between using another variable group and setting pipeline variables

Comment (digimaun) [reply .azure-devops/cleanup.yml:17]: The variable group will be configurable, we can assign the pipeline an existing var group or a new one dedicated for the clean up pipeline

Comment (vilit1) [reply .azure-devops/cleanup.yml:17]: done

**Comment by vilit1** [comment]
https://dev.azure.com/azureiotdevxp/aziotcli/_build/results?buildId=7369&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=73aacd12-d9e4-5437-3109-ca9de3b050de&l=92

**Review by digimaun — APPROVED** [review]

**Pull request merged**